CMakeLists.txt

```diff
@@ -27,6 +27,12 @@ if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE "Release")
 endif()
 
+# Optionally change stack size (e.g.: emcmake cmake . -Dbinaryen_STACK_SIZE=8388608)
+if(NOT binaryen_STACK_SIZE)
```

Comment:
Looks like the other options are in the form `BYN_` so please make this `BYN_STACK_SIZE`. Or, as this is only for binaryen.js/wasm, perhaps `BYN_WASM_STACK_SIZE`.